tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandlerTest.java

```diff
@@ -164,7 +173,7 @@ protected void setup() throws Exception {
         handler.ctx = mockCtx;
     }
 
-    @AfterMethod
+    @AfterClass
```

Comment:
Why change to `AfterClass`

Reply:
To reduce the test time because it took a lot of time in my local env. All methods use different topics and won't affect each other.

Reply:
It should avoid using the same topic between tests, which is dangerous. The following developers may fall into this trap.